it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java

```diff
@@ -32,17 +34,17 @@
 public class SonarScannerTestSuite {
 
   @ClassRule
-  public static final Orchestrator ORCHESTRATOR = createOrchestrator();
+  public static final OrchestratorRule ORCHESTRATOR = createOrchestrator();
 
-  private static Orchestrator createOrchestrator() {
+  private static OrchestratorRule createOrchestrator() {
     String sonarVersion = System
-      .getProperty("sonar.runtimeVersion", "LATEST_RELEASE[7.9]");
-    return Orchestrator.builderEnv()
+      .getProperty("sonar.runtimeVersion", "LATEST_RELEASE[9.9]");
+    return OrchestratorRule.builderEnv()
       .useDefaultAdminCredentialsForBuilds(true)
-      .setSonarVersion(
-        sonarVersion).addPlugin(MavenLocation
-        .of("org.sonarsource.sonarqube", "sonar-xoo-plugin",
-          sonarVersion)).build();
+      .setSonarVersion(sonarVersion)
+      .addBundledPluginToKeep("sonar-javascript")
+      .addPlugin(MavenLocation.of("org.sonarsource.sonarqube", "sonar-xoo-plugin", sonarVersion))
```

Comment:
Would it make sense to create one orchestrator that only includes `sonar-javascript` and another that only includes `sonar-xoo-plugin`? Might be over-optimization, but most tests don't need the js plugin, and your new test does not need the xoo plugin

Reply:
given the number of tests, I believe starting a second instance would take longer than the small overhead introduced by the extra plugin.